### PR TITLE
Improve and simplify SCSS classes and variables

### PR DESCRIPTION
**Pull request by janfaracik**
This is a small PR to improve and simplify some classes and variables used in Jenkins.

- The small border animation on clicked/focused elements is now a reusable mixin, which cuts down on code duplication and brings greater consistency to the newer Jenkins components
- The `item()` mixin (as described above, happy to hear suggestions for a better name) now uses variables for its background/shadow meaning that it's more theme-able than before
- Manage Jenkins SCSS has moved to the `/pages` folder
- Minor font size changes to the Manage Jenkins to bring more balance to the page + make it more responsive
- The plugin update bubble now pops in on page load for increased visibility

https://user-images.githubusercontent.com/43062514/174123791-71453922-7905-444c-9f93-a368ca543616.mov

- Some very minor color changes to increase vibrancy
- A new keyboard focus state for components using the `item()` mixin
image

### Proposed changelog entries

* N/A (_alternatively_ Minor color changes to UI components)

### Proposed upgrade guidelines

N/A

### Submitter checklist

- [ ] (If applicable) Jira issue is well described
- [ ] Changelog entries and upgrade guidelines are appropriate for the audience affected by the change (users or developer, depending on the change) and are in the imperative mood. [Examples](https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/changelogs/weekly.yml)
  * Fill-in the `Proposed changelog entries` section only if there are breaking changes or other changes which may require extra steps from users during the upgrade
- [ ] Appropriate autotests or explanation to why this change has no tests
- [ ] New public classes, fields, and methods are annotated with `@Restricted` or have `@since TODO` Javadoc, as appropriate. 
- [ ] For dependency updates: links to external changelogs and, if possible, full diffs

### Desired reviewers

@jenkinsci/sig-ux 

### Maintainer checklist

Before the changes are marked as `ready-for-merge`: 

- [ ] There are at least 2 approvals for the pull request and no outstanding requests for change
- [ ] Conversations in the pull request are over OR it is explicit that a reviewer does not block the change
- [ ] Changelog entries in the PR title and/or `Proposed changelog entries` are accurate, human-readable, and in the imperative mood
- [ ] Proper changelog labels are set so that the changelog can be generated automatically
- [ ] If the change needs additional upgrade steps from users, `upgrade-guide-needed` label is set and there is a `Proposed upgrade guidelines` section in the PR title. ([example](https://github.com/jenkinsci/jenkins/pull/4387))
- [ ] If it would make sense to backport the change to LTS, a Jira issue must exist, be a _Bug_ or _Improvement_, and be labeled as `lts-candidate` to be considered (see [query](https://issues.jenkins.io/issues/?filter=12146)).

<a href="https://gitpod.io/#https://github.com/jenkinsci/jenkins/pull/6667"></a>

### CODE REVIEW

**Review by timja — APPROVED** [review]
LGTM tested via gitpod and compared it with https://weekly.ci.jenkins.io

**Review by NotMyFault — APPROVED** [review]

**Comment by NotMyFault** [comment]
This PR is now ready for merge. We will merge it after ~24 hours if there is no negative feedback.
Please see the [merge process documentation](https://github.com/jenkinsci/jenkins/blob/master/docs/MAINTAINERS.adoc#merge-process) for more information about the merge process.
Thanks!

**Pull request merged**
